sh/configure_env.sh

```diff
@@ -0,0 +1,106 @@
+#
+# Author: Philip Deegan
+# Email : philip.deegan@polytechnique.edu 
+# Date  : 21 - September - 2017
+#
+# This script is to be included to setup variables to build tick
+#   It is done so to be cross platform and heterogeneous
+#
+# Variables
+#  $PY = "python" binary
+#  $PCONF = "python-config" binary
+#  $LDARGS=Linking arguments
+#  $LIB_POSTFIX= python library format 
+#  $LIB_POSTEXT= system library file extention
+#  $PROFILES= array of mkn profiles to build
+# 
+######################################################################
+
+set -e
+
```

Comment:
Maybe could you add comments such as
looking you python executable
looking for python-config, it will be needed to get Python link arguments

Reply:
If it fails there is an error - which is the important part

Reply:
Comments added